src/openhmd.c

```diff
@@ -491,6 +515,13 @@ void ohmd_set_default_device_properties(ohmd_device_properties* props)
 	props->ipd = 0.061f;
 	props->znear = 0.1f;
 	props->zfar = 1000.0f;
+	props->universal_distortion_k[0] = 0.0;
```

Comment:
Well with the previous shader it was possible to assume the DK2 specifications and it mostly would work ok-ish for other types of lenses (and just stretch them across regardless of resolution), I guess we can retry that with the current DK2 lens spec and see how far it goes.

The question though is if we need the custom frustum calculations that the oculus driver is doing at the moment in order to use the universal shader.